### DISCUSSION TITLE
Fix Missing Plots After 1.1.7

### DISCUSSION THREAD

**Discussion by D2BO** [Chia-Network/chia-blockchain#6592] [Support]
To fix this issue.
Edit config file
Search the word "Null"
replace the null where your wallet supposed to go with you wallet address.
See Below:

pool:
  logging: *XXXX
  network_overrides: *XXXX
  selected_network: mainnet
  xch_target_address: Null (**Wallet Address goes here**)

**Comment by BrandtH22** [comment]
@D2BO , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
